clients/vmdisplay/vm-shared.h

```diff
@@ -37,16 +37,14 @@
 
 #define SURFACE_NAME_LENGTH     64
 #define BIT(a)                  (1<<a)
-/*
- * If a buffer has been updated by a VM client app, the status field will have
- * this UPDATED bit set. Otherwise, it would have this bit cleared
- */
+
```

Comment:
In general, we don't allow C++ style comments anywhere in IAS code. So I cannot accept this change unless you revert back to C style comments again.

Reply:
hmm I saw // was mostly used in .cpp in example folder, such as many .cpp under clients. But if it's your policy, then I will remove those changes.